internal/forms/forms.go

```diff
@@ -1053,8 +1067,14 @@ func (b formMessageBuilder) scanTmplBuildMessage(tmplPath string) (MessageForm,
 		lineTmpl = fillPlaceholders(lineTmpl, placeholderRegEx, b.FormValues)
 		if strings.HasPrefix(lineTmpl, "Subject:") {
 			retVal.Subject = strings.TrimPrefix(lineTmpl, "Subject:")
-		} else {
+		} else if strings.HasPrefix(lineTmpl, "To:") {
+			retVal.To = strings.TrimPrefix(lineTmpl, "To:")
+		} else if strings.HasPrefix(lineTmpl, "Cc:") {
+			retVal.Cc = strings.TrimPrefix(lineTmpl, "Cc:")
+		} else if inBody {
 			retVal.Body += lineTmpl + "\n"
+		} else {
+			log.Printf("skipping unknown template line: '%s'", lineTmpl)
 		}
```

Comment:
I think this would be easier to read if was written as switch, like so:

```go
		switch {
		case strings.HasPrefix(lineTmpl, "Subject:"):
			retVal.Subject = strings.TrimPrefix(lineTmpl, "Subject:")
		case strings.HasPrefix(lineTmpl, "To:"):
			retVal.To = strings.TrimPrefix(lineTmpl, "To:")
		case strings.HasPrefix(lineTmpl, "Cc:"):
			retVal.Cc = strings.TrimPrefix(lineTmpl, "Cc:")
		case inBody:
			retVal.Body += lineTmpl + "\n"
		default:
			log.Printf("skipping unknown template line: '%s'", lineTmpl)
		}
```